### DISCUSSION TITLE
Submitting Vlang as an official Replit language.

### DISCUSSION THREAD

**Discussion by nac5604** [vlang/v#13208] [General]
Repl.it is a fantastic site for making quick apps or even big projects online. Currently there is a small template for running V code, but there's no official language support as of now. I'd personally like to use V alongside my other small projects on Replit; It would get me to use the language more since as of now I have no easy way of compiling and running V code.

Anyone else think this is a good idea?

**Comment by JalonSolov** [comment]
Anything that gets more use for V is a good thing, but this may have to wait until at least V 0.3.
